src/sider/dialogmoter/components/referat/Referat.tsx

```diff
@@ -466,19 +472,15 @@ const Referat = ({ dialogmote, mode }: ReferatProps): ReactElement => {
             </VStack>
           </VStack>
           <Alert
-            variant="warning"
+            variant="info"
```

Comment:
Har en antakelse om at de har blitt så vant til denne warningen at de kanskje ikke legger merke til hvis vi endrer litt på den 🤔 Nå er den jo på hvit bakgrunn (det er kanskje `inline`-flagget som gjør dette), kunne man vurdert å gjøre den "vanlig" for at den skal skille seg ut? 

Uansett så kan vi tracke lenke-klikk da, så det gir jo kanskje en viss indikasjon.